common/savefile.h

```diff
@@ -54,48 +54,89 @@ typedef SeekableReadStream InSaveFile;
  */
 class OutSaveFile: public WriteStream {
 protected:
-	WriteStream *_wrapped;
+	WriteStream *_wrapped; /*!< @todo Doc required. */
 
 public:
-	OutSaveFile(WriteStream *w);
+	OutSaveFile(WriteStream *w); /*!< Create a WriteStream instance @p w. */
```

Comment:
It doesn't create a WriteStream instance. It creates an `OutSaveFile` that uses the given `WriteStream` to actually write the data.

Technically, since `OutSaveFile` is itself a `WriteStream`, it does create a `WriteStream`, but not @p w. The `w` WriteStream already exists.